src/Controllers/HistoryViewerController.php

```diff
@@ -56,16 +64,88 @@ class HistoryViewerController extends LeftAndMain
     public function getClientConfig()
     {
         $clientConfig = parent::getClientConfig();
-
         foreach ($this->formNames as $formName) {
             $clientConfig['form'][$formName] = [
                 'schemaUrl' => $this->Link('schema/' . $formName),
             ];
         }
-
+        $clientConfig['endpoints'] = [
+            'revert' => $this->Link('api/revert'),
+            'read' => $this->Link('api/read'),
+        ];
         return $clientConfig;
     }
 
+    public function apiRead(HTTPRequest $request): HTTPResponse
+    {
+        $id = (int) $request->getVar('id');
+        $recordClass = $request->getVar('recordClass') ?? '';
```

Comment:
Should be consistent with naming and either have
- `recordClass` with `recordId`, or 
- class` with `id`

Reply:
Applies to `apiRevert` as well.

Reply:
I would say class with id, however there's loads of existing JS that uses recordClass including JS components that aren't in this PR, so I think we should just keep as is

Reply:
As discussed IRL - we should use `dataClass` for this endpoint.
The pre-existence of `recordClass` as the name of the variable in javascript it's relevant.

Reply:
Updated